### PR TITLE
PMM-3832 Add db parameter.

### PR DESCRIPTION
**Pull request by DexterHD**

### CODE REVIEW

File: api/managementpb/actions.proto [hunk 1]
```diff
@@ -95,6 +95,8 @@ message StartMySQLExplainActionRequest {
   string service_id = 2;
   // query example.
   string query = 3;
+  // db which will be used for explain query.
+  string db = 4;
```
Comment (AlekSi) [inline api/managementpb/actions.proto]: I think we should be a bit more tidy in both public API field names and documentation from now on.
```
// The database name for EXPLAIN query.
string database = 4;
```

Comment (AlekSi) [reply api/managementpb/actions.proto]: Note the second line too…

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-369/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-369/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-369/

**Pull request merged**